### ISSUE TITLE
`vue/multi-word-component-names` also applies to pages directory

### ISSUE THREAD

**Issue by nopeless** [vuejs/eslint-plugin-vue#2434]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version: 8.57.0** 
- **eslint-plugin-vue version: 9.23.0** 
- **Vue version: 3.4.21** 
- **Node version: v20.11.1** 
- **Operating System: win11 23h2** 

**Please show your full configuration:**

```json5
module.exports = {
  // ...
  parser: "vue-eslint-parser",
  parserOptions: {
    parser: "@typescript-eslint/parser",
  },
  extends: [
    // "@nuxt/eslint-config",
    // "plugin:prettier/recommended",
    "plugin:vue/vue3-recommended",
  ],
  // ...
};

```

**What did you do?**	

```vue
// index.vue
<template>
  <h1>hi</h1>
</template>
```

`1:1  error    Component name "index" should always be multi-word    vue/multi-word-component-names`

This should not trigger

**Repository to reproduce this issue**

https://stackblitz.com/edit/nuxt-starter-fvsxuf?file=.eslintrc.cjs

**Comment by waynzh** [comment]
You mean in a Nuxt project, the `pages` directory should not be checked? You might want to refer to the [`nuxt/eslint` config](https://github.com/nuxt/eslint/blob/main/packages-legacy/nuxt2-eslint-config/index.js#L150) which disabled this rule. 

Typically, this wouldn't be an issue as you can configure the [`ignores` property](https://eslint.vuejs.org/rules/multi-word-component-names.html#ignores-todo) to ignore certain component names in general project.

**Comment by FloEdelmann** [comment]
Note that the Nuxt config only disables the rule in certain directories using [`overrides`](https://eslint.org/docs/latest/use/configure/configuration-files#how-do-overrides-work), which you can also do manually in your project. So I'll close this as won't fix, because there already exists a method to disable this rule for certain directories.
